libkpod/config.go

```diff
@@ -26,8 +26,6 @@ const (
 	containerExitsDir   = oci.ContainerExitsDir
 	// DefaultMountsFile holds the default mount paths in the form "host:container"
 	DefaultMountsFile = "/usr/share/containers/mounts.conf"
-	// OverrideMountsFile holds the override mount paths in the form "host:container"
-	OverrideMountsFile = "/etc/containers/mounts.conf"
```

Comment:
Why do you want to remove this.  This eliminates the admins ability to override the defaults.  Admin are not supposed to edit files in /usr, which are provided by the distribution.  He is supposed to edit /etc, which would add content.

Reply:
I fear we have too many conf files for cri-o now. Can we have just one under /etc? Or even better make this a part of cri-o.conf itself.

Reply:
Well in a perfect packaging world, we would not be shipping /etc/crio/crio.conf but we would ship //usr/share/crio/crio.conf  and then tell the customer to copy that to /etc/crio/crio.conf to modify defaults.  At some point in the future we want to get to the point that a system reset back to defaults is
 rm -rf /etc /var
reboot

Reply:
I have no problem in removing the mounts.conf files and moving all content into crio.conf, but lets look at moving crio.conf into /usr/share/crio for the next release.  And allowing users to update it.

Change this PR to remove both mounts.conf and /etc/containers/mounts.conf and go with crio-conf.

Reply:
👍